src/pages/index.vue

```diff
@@ -1,49 +1,51 @@
 <template>
   <section>
-    <nav>
-      <h3>{{user && user.displayName}}</h3>
-      <button @click.stop.prevent="signInWithRedirect('google')">
-        Sign In With Google
-      </button>
-      <button @click.stop.prevent="signOut">
-        Sign Out
-      </button>
-    </nav>
+    <nav></nav>
+    <NavBar imgURL="rocket-insights-logo.svg" :user="user" />
```

Comment:
It's very easy to camelcase all the things when living in javascript land. But `imgURL` here is an attribute on markup, and that convention falls kebab casing (e.g. `img-url`).

It's also good to think about naming things. What is this image for? Is it an icon? Is it a background image in the navbar? Is it the logo? It's much better to be specific when you can about what a variable is so that the brain has to do less thinking. I would also point out that that's not a URL, it's not even a path, it's just the filename. You handle the path in the Navbar component itself.

Think of every component as being as dumb as possible. The only logic a component should be doing is when it needs to figure out something based on its own state. In this case, the Navbar doesn't need to figure out the image path for the logo, we can just supply it directly:
```jsx
<NavBar logo-path="~/assets/images.rocket-insights-logo.svg"...
```

The `logo-path` attribute doesn't even have to be a variable, and in this case, it should just be a string since it's never going to change.